index.html

```diff
@@ -19,7 +19,7 @@
         shortName: "vc-data-model",
 
         // subtitle for the spec
-        subtitle: "Expressing verifiable information on the Web",
+        subtitle: "Expressing information with verifiable provenance on the Web",
```

Comment:
I'm wondering if we can go with something more pedestrian here... for example: "Knowing where your information comes from on the Web" -- "Manufacturer labels for information on the Web" -- "Ensuring tamper resistant publication of information on the Web."

Reply:
"Verifiably expressing the origin and transfer history of information on the Web"?

Reply:
(Maybe we can add a bit of informative text to the body somewhere, defining information provenance as its origin and transfer history, and linking out to PROV-O?)

Reply:
Since the title is "Verifiable Credentials Data Model v1.0" could the subtitle be "data structures for verifiable information" ?

Reply:
Except it's not verifiable _information_.

I still don't think provenance is that much of a mystery, and _that's_ what it is we're establishing -- a way for a Reader/Verifier to be cryptographically certain that the declared Asserter/Issuer made the Assertions/Claims found in the Verifiable Credential.

Reply:
"data structures for information provenance" ?

Reply:
Looks a lot like my suggestion above, i.e., "Expressing information with verifiable provenance [on the Web]" where the last phrase could be kept or removed, since I don't think VCs are actually limited to Web use...